ksql-parser/src/main/java/io/confluent/ksql/parser/SqlFormatter.java

```diff
@@ -458,7 +457,7 @@ protected Void visitShowPartitions(final ShowPartitions node, final Integer cont
     }
 
     @Override
-    protected Void visitShowFunctions(final ShowFunctions node, final Integer context) {
+    protected Void visitShowFunctions(final ListFunctions node, final Integer context) {
```

Comment:
I've renamed `ShowFunctions` to `ListFunctions` as all the other non executable list DDL commands use list, not show. So just improving consistency.